# try:
#     number = int(input("Введите целое число: "))
#     print("Число которое Вы ввели:", number)
# except ValueError:
#     print("Приоброзование числа прошло неудачно")
# print("Програма завершена!")

try:  # попытка !
    a = int(input("Ведите число a: "))
    b = int(input("Ведите число b: "))
    print("Результа деления а / b =", a / b)
except ValueError:  # При ошибке не бьет ошибку в консоле и красиво выводит програму
    print("Числа которые Вы вводите должны быть целыми(int)")
except ZeroDivisionError:  # ошибка которая при деление на 0
    print("Попытка деления на ноль само собой невозможно!")
except Exception:  # другире ошибки, подчеркивает потому что и так все возможные перечислили!
    print("Общее исключение!")
finally:  # блок когда который исполняется в любом случае
    print("Блок try завершил выполнение!")
print("Програма завершена!")

